src/components/inputs/select.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { SingleSelectField, SingleSelectOption } from '@dhis2/ui'
+
+export const Select = ({
+    onChange,
+    label,
+    name,
+    value,
+    disabled,
+    options,
+    inputWidth,
+}) => (
```

Comment:
This component looks good, it has all the props explicitly!  It should also have PropTypes, but this is a good model for the other Select components